City-of-Pori/pori-kada
# Feature/pori 66 page styles

To test:

1. drush updb -y && drush fra -y && drush cc all
2. Go to edit page that is on the first level of navigation and make sure that they have the "Theme main page" checkbox checked under "Widgets" tab.
3. Now go to edit some page that is NOT on the first level of navigation. Make sure there isn't the just mentioned checkbox checked.
4. Make sure the page has lead paragraph and some keywords and maybe some subpages.
5. Make sure the layout looks nice and that it looks the as defined in the layout documents (I did adjust some margins to smaller tho, but you know what I mean)

NOTICE: Additional information block will come in a different story so don't search for that title in the layout. It is not there. Also make sure that the PHP that I wrote to preprocess_html is ok!!

Layouts: https://drive.google.com/drive/u/0/folders/0B7ZHsug7PcEDVDBNTG51a0RIMzQ
